app/controllers/posts_controller.rb

```diff
@@ -22,4 +22,10 @@ def create
       render :new
     end
   end
```

Comment:
- [ ] Please kindly use **CanCanCan** for Post authorization in your `Post Controller`, some like so: 

```ruby
  class PostsController < ApplicationController
    load_and_authorize_resource

    ...
  end
```